scripts/tests/test_zkai_ezkl_statement_envelope_benchmark.py

```
@@ -0,0 +1,126 @@
+
+        self.assertEqual(
+            envelope["statement_commitment"],
+            BENCH.statement_commitment(envelope["statement"]),
+        )
+
+    def test_raw_proof_only_accepts_metadata_relabeling_with_fake_verifier(self) -> None:
+        _category, envelope = BENCH.mutated_envelopes()["model_id_relabeling"]
+
+        BENCH.verify_proof_only(
+            envelope,
+            pathlib.Path("/tmp/nonexistent-srs-not-read-by-fake"),
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/nonexistent-srs-not-read-by-fake"

(S108)